ci/change-runtime-settings.py

```diff
@@ -0,0 +1,72 @@
+        if option.startswith('+'):
+            return "\n{}=".format(option).join(value.split('\n'))
+        return value
+
+class CustomWriter:
+    output_file = None
+    def __init__(self, new_output_file):
+        self.output_file = new_output_file
+    def write(self, what):
+        self.output_file.write(what.replace('\n\t','\n'))
+        
+
+# modify runtime settings before cook
+def change_runtime_settings(project_home):
+    defaultengine = os.path.join(project_home, 'Game', 'Config', 'DefaultEngine.ini')
+    config = configparser.ConfigParser(strict=False, dict_type=MultiOrderedDict, interpolation=CustomInterpolation())
+    config.optionxform = lambda option: option  # preserve case for letters
+    config.read(defaultengine)
+
+    IOSRuntimeSettings = '/Script/IOSRuntimeSettings.IOSRuntimeSettings'
+    if not config.has_section(IOSRuntimeSettings):
+        config.add_section(IOSRuntimeSettings)
+    
+    # for ios runtime settings
+    AdditionalPlistData = '<key>CFBundleURLTypes</key><array><dict><key>CFBundleURLName</key><string></string><key>CFBundleTypeRole</key><string>Editor</string><key>CFBundleURLSchemes</key><array><string>firebase-game-loop</string></array></dict></array>'
+    config[IOSRuntimeSettings]['AdditionalPlistData'] = AdditionalPlistData
+    config[IOSRuntimeSettings]['BundleIdentifier'] = 'io.improbable.unrealgdkdemo'
+    config[IOSRuntimeSettings]['MobileProvision'] = ''
+    config[IOSRuntimeSettings]['BundleDisplayName'] = 'UnrealGDK Shooter'
+    config[IOSRuntimeSettings]['BundleName'] = 'unrealgdkshooter'
+    config[IOSRuntimeSettings]['bSupportsITunesFileSharing'] = 'True'
+    config[IOSRuntimeSettings]['bGeneratedSYMFile'] = 'True'
+    config[IOSRuntimeSettings]['bGeneratedSYMBundle'] = 'True'
+    config[IOSRuntimeSettings]['bGenerateCrashReportSymbols'] = 'True'
+    config[IOSRuntimeSettings]['bEnableRemoteNotificationsSupport'] = 'False'
+    config[IOSRuntimeSettings]['bEnableAdvertisingIdentifier'] = 'False'
+    config[IOSRuntimeSettings]['bEnableCloudKitSupport'] = 'False'
+    
+    # for Android extra activity settings
+    AndroidRuntimeSettings = '/Script/AndroidRuntimeSettings.AndroidRuntimeSettings'
+    if not config.has_section(AndroidRuntimeSettings):
+        config.add_section(AndroidRuntimeSettings)
+    ExtraActivitySettings = '<intent-filter><action android:name="com.google.intent.action.TEST_LOOP"/><category android:name="android.intent.category.DEFAULT"/><data android:mimeType="application/javascript"/></intent-filter>'  
+    config[AndroidRuntimeSettings]['ExtraActivitySettings'] = ExtraActivitySettings
+    config[AndroidRuntimeSettings]['bSupportAdMob'] = 'False'
+    config[AndroidRuntimeSettings]['bPackageDataInsideApk'] = 'True'
+    with open(defaultengine,'w') as fw:
+        config.write(CustomWriter(fw), space_around_delimiters=False)
+
+
+if __name__ == "__main__":
+    change_runtime_settings(sys.argv[1])
```

Comment:
I'd prefer if we define the input args for future maintainability. 
```suggestion
	# Path to the UnrealGDKExampleProject folder.
	project_home = sys.argv[1]
    change_runtime_settings(project_home)
```